assets/airlines/fle.json

```diff
@@ -0,0 +1,15 @@
+{
+    "icao": "fle",
+    "name": "Flair Airlines",
+    "callsign": {
+        "name": "Flair",
+        "length": 3
+    },
+    "fleets": {
+        "default": [
+            ["B734", 5],
+            ["D328", 1],
```

Comment:
I think the D328 and C340 were retired

Reply:
They still own them though: https://www.flightradar24.com/data/aircraft/flair-airlines-f8-fle

Reply:
fair enough, disregard then 👍